src/main/java/frc/robot/commands/TeleopDrivebaseDefaultCommand.java

```diff
@@ -18,15 +18,19 @@ public class TeleopDrivebaseDefaultCommand extends CommandBase {
     private DoubleSupplier strafeSup;
     private DoubleSupplier rotationSup;
     private BooleanSupplier robotCentricSup;
+    private BooleanSupplier slowModeSupplier;
 
-    public TeleopDrivebaseDefaultCommand(DrivebaseSubsystem s_Swerve, DoubleSupplier translationSup, DoubleSupplier strafeSup, DoubleSupplier rotationSup, BooleanSupplier robotCentricSup) {
+    public TeleopDrivebaseDefaultCommand(DrivebaseSubsystem s_Swerve, DoubleSupplier translationSup, DoubleSupplier strafeSup, DoubleSupplier rotationSup, BooleanSupplier robotCentricSup, BooleanSupplier slowModeSupplier) {
```

Comment:
Is this supposed to be in this PR, doesn't seem to affect the robot leveling? Might just be a symptom of merging the updated version of the drivebase, in which case that is fine.